tools/replay/CMakeLists.txt

```diff
@@ -37,14 +37,30 @@ target_sources(gfxrecon-replay
                     $<$<BOOL:WIN32>:${CMAKE_SOURCE_DIR}/version.rc>
 )
 
-target_include_directories(gfxrecon-replay PUBLIC ${CMAKE_BINARY_DIR})
+if (WIN32)
+    target_include_directories(gfxrecon-replay PUBLIC ${CMAKE_BINARY_DIR} ../../external/json/win64/include)
```

Comment:
I think `${CMAKE_BINARY_DIR} ../../external/json/win64/include` is the wrong directory.
It should be `${CMAKE_SOURCE_DIR}/external/json/linux/include`
Same for the else case